packages/core/src/types.ts

```diff
@@ -219,6 +229,32 @@ export type Plot = Scatterplot | Histogram;
 
 export type Bookmark = { id: string; label: string; size: number }
 
+/**
+* Represents the alternative text for an Upset plot.
+*/
+export type AltText = {
+  /**
+  * The long description for the Upset plot.
+  */
+  longDescription: string;
+
+  /**
+  * The short description for the Upset plot.
+  */
+  shortDescription: string;
+
+  /**
+  * The technique description for the Upset plot.
+  */
+  techniqueDescription?: string;
```

Comment:
What is the technique description? Does this come from the alt-txt library?

Reply:
Yes, this comes from alttext but isn't implemented yet. I'm not sure what Eliza's plans are but I made this forward compatible because that field is included in the JSON response.